eng/packages/http-client-csharp-mgmt/generator/Azure.Generator.Management/src/Utilities/PropertyHelpers.cs

```diff
@@ -87,6 +87,11 @@ public static MethodBodyStatement BuildGetter(bool? includeGetterNullCheck, Prop
             // For collection types, we initialize the internal property if it's null and return the inner property.
             if (innerProperty.Type.IsCollection && internalProperty.WireInfo?.IsRequired == true)
             {
+                if (internalProperty.Body.HasSetter == false)
```

Comment:
is the type of `HasSetter` property `bool?`? if so I think we should open an issue for MTG because this does not make any sense - a property with or without setter, this fact does not have the third state.
if it is bool, I think we should simplify this to
```suggestion
                if (!internalProperty.Body.HasSetter)
```

Reply:
changed to  if (!internalProperty.Body.HasSetter)